apps/nrm/behaviours/save-form-session.js

```diff
@@ -54,7 +54,9 @@ module.exports = superclass => class extends superclass {
           return res.redirect('/nrm/save-and-exit');
         }
 
-        if (req.path.match(/\/change$/)) {
+        const noEditContinue = !req.form.options.continueOnEdit;
+
+        if (req.sessionModel.get('redirect-to-reports') && noEditContinue) {
```

Comment:
can you just explain what's going on here?

Reply:
a flag is set to redirect back to the reports in progress. This is to ensure it doesn't redirect to the latest question in progress or the check your answers page at the moment. It's intended to take the user back to whence they came.

Reply:
👍